sympy/plotting/experimental_lambdify.py

```diff
@@ -239,6 +240,9 @@ def __call__(self, args):
 
 
 def experimental_lambdify(*args, **kwargs):
+    if kwargs.get('use_python_math', None) is True:
+        if kwargs.get('complex_wrap_evalf', None):
+            kwargs['complex_wrap_evalf']=False
     l = Lambdifier(*args, **kwargs)
```

Comment:
does Lambdifier pass the kwargs on to `__call__`?

Reply:
It does not passed it because when i checked kwargs in ```__call__``` function it was empty dict.

Sorry for delayed answer ,it was due to lack of internet while travelling.